ieeetran/footer.tex

```diff
@@ -5,10 +5,27 @@
 \printbibliography
 \fi
 
+\appendices
```

Comment:
Digging a bit further in, the command is defined in the `IEEEtran.cls` file, and is manipulating a bunch of internal counters and reconfiguring the `section` family of commands. As far as I can tell, this is harmless if none of the sections are actually in use.